### PR TITLE
Updating javascript dependancies on dated reminders

### PR DESCRIPTION
**Pull request by Gicheha**

### CODE REVIEW

File: interface/main/dated_reminders/dated_reminders_add.php [hunk 1]
```diff
@@ -172,14 +172,13 @@
 <html>
   <head>
     <title><?php echo xlt('Send a Reminder') ?></title>
+    <?php call_required_libraries(array("jquery-min-3-3-1","font-awesome", "iziModalToast"));?>
```
Comment (muarachmann) [inline interface/main/dated_reminders/dated_reminders_add.php]: This looks nicer and cleaner @Gichecha. Hope you enjoyed the git tutorial i gave you. Welcome on board. I am approving this. Please delete the other PR

Comment (muarachmann) [reply interface/main/dated_reminders/dated_reminders_add.php]: Oh please edit your PR title. I know you are super excited it worked. Lets keep things professional :)

Comment (Gicheha) [reply interface/main/dated_reminders/dated_reminders_add.php]: Done...thanks

**Comment by muarachmann** [comment]
@Gicheha please can you rename it to`Updating javascript dependancies on dated reminders` so it ties with your reported issue? Also consider deleting #1469

**Pull request merged**